main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Main.java

```diff
@@ -69,6 +70,9 @@ public static void main(String[] argv) {
         } catch (IOException exception) {
             exception.printStackTrace();
         }
+
+        final long endMillis = System.currentTimeMillis();
+        System.out.println(String.format("Validation took %.3f seconds", (endMillis - startMillis) / 1000.0));
```

Comment:
What do you think about leveraging `java.util.concurrent.TimeUnit`? The maths here are relatively simple but we could use this library to improve clarity (conversion from milliseconds to seconds)
```suggestion
        System.out.println(String.format("Validation took %.3f seconds", TimeUnit.MILLISECONDS.toseconds(endMillis - startMillis));
```

Reply:
Agree with ☝️ , although the units should be in nanos (see my other comment).

Reply:
Unfortunately, these converters in `TimeUnit` are not as good as they should be. They return `long` instead of `double` which means that we would get 0 seconds for most of the small feeds and this measurement would be pointless.

I agree to use `System.nanoTime()`, absolutely, but I insist on keeping division by double here.

Reply:
We have a wonderful function absl::FDivDuration() that would do what we need here, but this is for C++ :)

Reply:
>Unfortunately, these converters in TimeUnit are not as good as they should be. They return long instead of double which means that we would get 0 seconds for most of the small feeds and this measurement would be pointless.

Ah, yes, you're right, I had forgotten about this. We did the same thing in the GTFS-RT validator.